tests/test-samba-ad-server-kubernetes.sh

```diff
@@ -10,6 +10,14 @@ kubectl exec "${podname}" -- samba-tool domain info 127.0.0.1
 [ $? -eq 0 ] || _error "Error listing domain info"
 echo
 
+source ${SCRIPT_DIR}/test-deploy-ad-member.sh
+
+kubectl exec "${podname}" -c "smb" -- smbclient -N -L 127.0.0.1
+kubectl exec "${podname}" -c "winbind" -- wbinfo -t
+kubectl exec "${podname}" -c "smb" -- wbinfo -t
```

Comment:
I was momentarily confused by two "wbinfo -t" but then I realized it makes sense as the container running smbd needs to communicate with winbind and `wbinfo -t` can demonstrate that. So, perhaps a comment will save future readers like me from that confusion.

Reply:
exactly, that's the point. :-)
Will add a comment.

Reply:
done